packages/components/psammead-section-label/src/__snapshots__/index.test.jsx.snap

```diff
@@ -1657,6 +1657,272 @@ exports[`SectionLabel With bar With plain title should render correctly with exp
 </div>
 `;
 
+exports[`SectionLabel With heading overriden should render a strong element instead of an h2 1`] = `
+.c2 {
+  display: -webkit-box;
+  display: -webkit-flex;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-flex-direction: column;
+  -ms-flex-direction: column;
+  flex-direction: column;
+}
+
+.c3 {
+  display: -webkit-box;
+  display: -webkit-flex;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-flex-flow: row nowrap;
+  -ms-flex-flow: row nowrap;
+  flex-flow: row nowrap;
+  -webkit-box-pack: justify;
+  -webkit-justify-content: space-between;
+  -ms-flex-pack: justify;
+  justify-content: space-between;
+  min-height: 2.75rem;
+  -webkit-align-items: baseline;
+  -webkit-box-align: baseline;
+  -ms-flex-align: baseline;
+  align-items: baseline;
+}
+
+.c4 {
+  font-size: 1.125rem;
+  line-height: 1.375rem;
+  font-family: ReithSans,Helvetica,Arial,sans-serif;
+  font-weight: 400;
+  font-style: normal;
+  background-color: #FDFDFD;
+  margin: 1rem 0;
+  padding-right: 0.5rem;
+  display: -webkit-box;
+  display: -webkit-flex;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-align-items: center;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+}
+
+.c0 {
+  position: relative;
+  z-index: 0;
+  color: #3F3F42;
+  margin-top: 2rem;
+}
+
+.c1 {
+  margin: 0;
+  padding: 0;
+}
+
+@media (min-width:37.5rem) {
+  .c3 {
+    -webkit-align-items: stretch;
+    -webkit-box-align: stretch;
+    -ms-flex-align: stretch;
+    align-items: stretch;
+  }
+}
+
+@media (min-width:20rem) and (max-width:37.4375rem) {
+  .c4 {
+    font-size: 1.25rem;
+    line-height: 1.5rem;
+  }
+}
+
+@media (min-width:37.5rem) {
+  .c4 {
+    font-size: 1.5rem;
+    line-height: 1.75rem;
+  }
+}
+
+@media (min-width:37.5rem) {
+  .c4 {
+    margin: 0;
+  }
+}
+
+@media (min-width:37.5rem) {
+  .c4 {
+    padding-right: 1rem;
+  }
+}
+
+@media (min-width:37.5rem) {
+  .c0 {
+    margin-top: 1.5rem;
+  }
+}
+
+@media (min-width:63rem) {
+  .c0 {
+    margin-bottom: 1.5rem;
+  }
+}
+
+<div
+  class="c0"
+>
+  <strong
+    class="c1"
+  >
+    <span
+      class="c2"
+    >
+      <span
+        class="c3"
+      >
+        <span
+          class="c4"
+          dir="ltr"
+          id="test-section-label"
+        >
+          This is text in a SectionLabel.
+        </span>
+      </span>
+    </span>
+  </strong>
+</div>
+`;
+
+exports[`SectionLabel With heading overriden to be a strong element should render correctly as a plain title with no bar 1`] = `
```

Comment:
whats the difference in this to the example above?

Reply:
@greenc05 none actually, just a change in the test description title.